data/names/ru.json

```diff
@@ -8679,7 +9296,1005 @@
       "Янира",
       "Янита",
       "Ясмин",
-      "Яхайра"
+      "Яхайра",
+      [ "Аарон", "Aaron" ],
+      [ "Августин", "Augustine" ],
+      [ "Агнес", "Agnes" ],
+      [ "Адалин", "Adaline" ],
+      [ "Адам", "Adam" ],
+      [ "Аделл", "Adell" ],
+      [ "Аджа", "Aja" ],
+      [ "Адриан", "Adrian" ],
+      [ "Азалия", "Azalee" ],
+      [ "Ай", "Ai" ],
+      [ "Айдэ", "Aide" ],
+      [ "Айко", "Aiko" ],
+      [ "Айриш", "Irish" ],
+      [ "Акико", "Akiko" ],
+      [ "Алан", "Alane" ],
+      [ "Алейда", "Aleida" ],
+      [ "Александр", "Alexander" ],
+      [ "Ален", "Alaine" ],
+      [ "Алеша", "Aleshia" ],
+      [ "Алида", "Alida" ],
+      [ "Алика", "Alica" ],
+      [ "Даймонд", "Diamond" ],
+      [ "Альберт", "Albert" ],
+      [ "Альма", "Alma" ],
+      [ "Ампаро", "Amparo" ],
+      [ "Ан", "An" ],
+      [ "Ангелена", "Angelena" ],
+      [ "Анджелес", "Angeles" ],
+      [ "Анджелин", "Angelyn" ],
+      [ "Анджело", "Angelo" ],
+      [ "Эндрю", "Andrew" ],
+      [ "Анжеле", "Angele" ],
+      [ "Анжелита", "Angelita" ],
+      [ "Анжелия", "Angelia" ],
+      [ "Аннали", "Annalee" ],
+      [ "Анналиса", "Annalisa" ],
+      [ "Аннамае", "Annamae" ],
+      [ "Аннемари", "Annemarie" ],
+      [ "Аннетта", "Annetta" ],
+      [ "Аннмари", "Annmarie" ],
+      [ "Антионетта", "Antionette" ],
```

Comment:
Should probably be `Антуанетта`, like [`Мария-Антуанетта`](https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D1%80%D0%B8%D1%8F-%D0%90%D0%BD%D1%82%D1%83%D0%B0%D0%BD%D0%B5%D1%82%D1%82%D0%B0), the one who supposedly, but with no evidence to back it up, said "Let them eat cake". Otherwise, with no association with the princess (and later queen), it would be `Онтуанетт`, to follow up on the name's clear French roots.